config.toml

```diff
@@ -3,7 +3,7 @@ relativeurls = false
 languageCode = "en-us"
 title = "Tidyverse"
 description = "An opinionated collection of R packages designed for data science."
-googleAnalytics = ""
+googleAnalytics = "UA-115082821"
```

Comment:
It might be less confusing to remove this line altogether since our custom theme no longer uses it

Reply:
Oh…ok. I thought GA and GTM were actually different. Will change.

Reply:
Oh maybe they are?

Reply:
Oh they are different. Do we need both?